modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavascriptClosureAngularClientCodegen.java

```diff
@@ -46,7 +61,8 @@ public JavascriptClosureAngularClientCodegen() {
         hideGenerationTimestamp = Boolean.TRUE;
 
         supportsInheritance = false;
-        setReservedWordsLowerCase(Arrays.asList("abstract",
+        // TODO Are those case sensitive or insensitive
+        super.registerReservedWordsCaseInsensitive(Arrays.asList("abstract",
```

Comment:
case sensitive

Reply:
Resolved in commit [11ff0928](https://github.com/OpenAPITools/openapi-generator/commit/11ff0928fd16a8a9342c43b65026e1c4dc973d3d)